### PR TITLE
Improved swiping through notifications behaviour

### LINKED ISSUE #14919: Improve swiping through notifications behaviour
When swiping between the screens for e.g. swiping through notifications on the Notification tab, the foreground and the background screens are not distinguishable (See https://github.com/wordpress-mobile/WordPress-Android/pull/14906#pullrequestreview-690723314).

Screenshot 2021-06-24 at 9 25 03 AM

Giving such screens an elevation doesn't look great either (See https://github.com/wordpress-mobile/WordPress-Android/pull/14906#issuecomment-867333493). 

device-2021-06-24-102355

The goal of this task to explore our options to further improve this swiping behavior.

### PR DESCRIPTION
**Pull request by ssinha2103**
Partially fixes #14919

Screenshot 2021-08-29 at 9 41 19 PM

To test:

## Regression Notes
1. Potential unintended areas of impact

2. What I did to test those areas of impact (or what existing automated tests I relied on)

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/src/main/res/layout/notifications_detail_activity.xml [hunk 1]
```diff
@@ -14,33 +34,17 @@
...
     <org.wordpress.android.widgets.WPViewPager
         android:id="@+id/viewpager"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior" />
-
-    <ProgressBar
-        android:id="@+id/progress_loading"
         android:layout_width="wrap_content"
+        android:background="@color/gray_5"
```
Comment (ssinha2103) [inline WordPress/src/main/res/layout/notifications_detail_activity.xml]: Added it to give the transition a background colour for distinction between notification

File: WordPress/src/main/res/drawable/back.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle" >
+    <solid android:color="@android:color/white" />
+    <stroke android:width="5dip" android:color="@color/black"/>
```
Comment (ashiagr) [inline WordPress/src/main/res/drawable/back.xml]: ❓ : Do we need this `5dp` `black` border? Can't seem to find it in the transition.

File: WordPress/src/main/res/layout/notifications_list_fragment.xml [hunk 3]
```diff
@@ -50,6 +50,7 @@
         android:layout_height="match_parent"
         android:layout_marginTop="@dimen/toolbar_height"
         android:gravity="center"
+        android:background="@drawable/back"
```
Comment (ashiagr) [inline WordPress/src/main/res/layout/notifications_list_fragment.xml]: ~~Suggestion: We can rename `back` with a generic name like `bg_rectangle_white_stroke_black` to be consistent with other color drawables. But wait to rename it till the color is decided.~~

Not required due to https://github.com/wordpress-mobile/WordPress-Android/pull/15253#discussion_r698237365

**Review by ashiagr — COMMENTED** [review]
Thank you for working on this @ssinha2103!

Looks nice in the light mode. Needs some adjustments in the dark mode. Instead of white, can you try a darker background in the dark mode?

File: WordPress/src/main/res/drawable/back.xml (hunk 2 above)
Comment (ssinha2103) [reply WordPress/src/main/res/drawable/back.xml]: Actually we don't need the whole file itself, I used it to try something , but didn't work well, but I forgot to remove the file.

Comment (ashiagr) [reply WordPress/src/main/res/drawable/back.xml]: Yeah, I noticed it affected another part of the screen, let's remove it.

File: WordPress/src/main/res/layout/notifications_detail_activity.xml [hunk 4]
```diff
@@ -14,33 +34,17 @@
...
     <org.wordpress.android.widgets.WPViewPager
         android:id="@+id/viewpager"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior" />
-
-    <ProgressBar
-        android:id="@+id/progress_loading"
         android:layout_width="wrap_content"
+        android:background="@color/placeholder"
```
Comment (ssinha2103) [inline WordPress/src/main/res/layout/notifications_detail_activity.xml:45]: Changed to placeholder and committed.

Comment (ssinha2103) [reply WordPress/src/main/res/layout/notifications_detail_activity.xml:45]: Screenshot 2021-08-30 at 1 20 06 PM
Screenshot 2021-08-30 at 1 20 36 PM
Modified it for both Light And Dark Mode.

File: WordPress/src/main/res/drawable/back.xml (hunk 2 above)
Comment (ashiagr) [reply WordPress/src/main/res/drawable/back.xml]: Hey, I don't see `back.xml` removed yet. Let's not mark it resolved till then.

File: WordPress/src/main/res/layout/notifications_detail_activity.xml (hunk 4 above)
Comment (ashiagr) [reply WordPress/src/main/res/layout/notifications_detail_activity.xml:45]: Thank you! Let's wait till we get a design review.

File: WordPress/src/main/res/layout/notifications_list_fragment.xml [hunk 5]
```diff
@@ -50,7 +50,6 @@
         android:layout_height="match_parent"
         android:layout_marginTop="@dimen/toolbar_height"
         android:gravity="center"
-        android:background="@drawable/back"
```
Comment (ssinha2103) [inline WordPress/src/main/res/layout/notifications_list_fragment.xml]: Hii @ashiagr , I removed the back.xml, should we resolve the issue now ?

File: WordPress/src/main/res/drawable/back.xml (hunk 2 above)
Comment (ssinha2103) [reply WordPress/src/main/res/drawable/back.xml]: Hii @ashiagr , I removed it now. Please check.

File: WordPress/src/main/res/layout/notifications_detail_activity.xml (hunk 4 above)
Comment (ashiagr) [reply WordPress/src/main/res/layout/notifications_detail_activity.xml:45]: As discussed, we're merging this and we'll capture [suggested animation](https://github.com/wordpress-mobile/WordPress-Android/issues/14919#issuecomment-908503715) in a separate task.

**Review by ashiagr — APPROVED** [review]

**Pull request merged**